tests/sandbox/content_types/test_content_negotiation.py

```diff
@@ -29,7 +29,19 @@
 @pytest.mark.parametrize("method", METHODS)
 def test_application_response_type(nhsd_apim_proxy_url, accept_headers, method):
     """
-    .. py:function:: Test content negotiation
+    .. py:function:: Scenario: A user submitting a request with a valid accept header \
+        receives a response containing the expected accept header
+
+        | **Given** the user provides a valid accept header
+        | **When** the user submits a request
+        | **Then** the response returns the expected accept header
```

Comment:
`**Then** the response returned is in the format requested by the user`
We're asserting that the response content type is correct for the given `accept` header